beep-beep/jobs.md

```diff
@@ -0,0 +1,1136 @@
+- `priority` - Similar to the same option in other drivers. This is queue 
+  default priority for for each task pushed into this queue if the priority 
+  value for these tasks was not explicitly set.
+
+- `tube_priority` - The value for specifying the priority within Beanstalk is
+  the internal priority of the server. The value should not exceed `int32` size.
+
+- `tube` - The name of the inner "tube" specific to the Beanstalk driver.
+
+### SQS Driver
+
+[Amazon SQS (Simple Queue Service)](https://aws.amazon.com/sqs/) is an
+alternative queue server also developed by Amazon and is also part of the AWS
+service infrastructure.
+
+However, unlike Beanstalk, it is located entirely on Amazon servers, so the
+connection configuration will differ from the one we are already used to for
+AMQP and Beanstalk. In order to get acquainted with how to create a SQS server, 
+just use the [ready-made documentation](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-configuring.html).
+
+Please also note that although installation as a server using AWS services is
+permissible, however, it is possible to install this server locally by
+downloading the corresponding executable file from Amazon servers and running 
+it using Java (you will also need the installed Java Runtime Environment):
+
+```sh
+wget https://s3-eu-west-1.amazonaws.com/softwaremill-public/elasticmq-server-1.2.0.jar
+
+// Where "custom.conf" is your own config file
+java -Dconfig.file=custom.conf -jar elasticmq-server-1.2.0.jar
+```
+
+After you have created the SQS server, you need to specify the following 
+connection settings in `sqs` configuration settings:
+
+```yaml
+sqs:
+  # Required AccessKey ID.
+  # Default: empty
+  key: access-key
+
+  # Required secret access key.
+  # Default: empty
+  secret: api-secret
+
+  # Required AWS region.
+  # Default: empty
+  region: us-west-1
+
+  # Required AWS session token.
+  # Default: empty
+  session_token: test
+
+  # Required AWS SQS endpoint to connect.
+  # Default: http://127.0.0.1:9324
+  endpoint: http://127.0.0.1:9324
+```
+
+> Please note that although each of the sections contains default values, it is
+> marked as "required". This means that in almost all cases they are required to
+> be specified in order to correctly configure the driver.
+
+After you have configured the connection - you should configure the queue that
+will use this connection:
+
+```yaml
+sqs:
+  # SQS connection configuration...
+
+jobs:
+  pipelines:
+    # Required section.
+    # Should be "sqs" for the Amazon SQS driver.
+    driver: sqs
+    
+    # Optional section.
+    # Default: 10
+    prefetch: 10
+    
+    # Optional section.
+    # Default: 0
+    visibility_timeout: 0
+    
+    # Optional section.
+    # Default: 0
+    wait_time_seconds: 0
+    
+    # Optional section.
+    # Default: default
+    queue: default
+    
+    # Optional section.
+    attributes:
```

Comment:
Put at least 1 attribute like a sample for a user.

Reply:
Done. However, example attribute definition is found below in the description of this section.